### PR TITLE
Added a directory structur for GML XML sample documents

### LINKED ISSUE #4: Add a directory for GML XML examples
We will need sample GML XML documents for testing purposes.

### PR DESCRIPTION
**Pull request by highsource**
Fixes #4.

### CODE REVIEW

File: examples/GML/3.1.1/position-00.gml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<gml:position xmlns:gml="http://www.opengis.net/gml">
```
Comment (derhuerst) [inline examples/GML/3.1.1/position-00.gml:1]: Do GML files need the `<?xml version="1.0" encoding="UTF-8"?>` prefix?

**Comment by derhuerst** [comment]
While I agree that we need some "end-to-end" tests, I think the individual tests should not read & parse XML (after all this is not what this lib does), but [work with a virtual tree like at `parse-gml-polygon`](https://github.com/derhuerst/parse-gml-polygon/blob/9845528b0aed6a8b58f6680b1c82e5c4b454f3fb/test.js#L61-L65). This would make the tests more specific (they won't fail if the XML parser fails) and more portable.

Let's move the `examples/GML` directory to `test/fixtures` and keep the "end to end test files" to an absolute minimum – a few files per GML version – and use virtual-tree-based tests for all individual test cases.

File: examples/GML/3.1.1/position-00.gml (hunk 1 above)
Comment (highsource) [reply examples/GML/3.1.1/position-00.gml:1]: No, not necessarily.

**Comment by highsource** [comment]
My primary intention was not E2E test files but more a collection of examples to be used in development to see what certain concept or encoding looks in GML.

**Comment by derhuerst** [comment]
There is no other place with a collection of simple GML files? If so, I'd prefer to keep this repo small. If not, let's merge it.

**Comment by highsource** [comment]
I think it's useful to keep them in the project. Later we'll migrate them to E2E tests. At the moment I just want to keep them somewhere.

**Pull request merged**